README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
+- [Headphones](https://thenounproject.com/search/?q=headphones&i=16961) icon Molly Bramlet, from the Noun Project
```

Comment:
Docs contradict themselves: https://thenounproject.zendesk.com/hc/en-us/articles/200509928-How-do-I-give-creators-credit-in-my-work-

According to the 1st sentence of the 2nd para, the current style is OK. But the example at the bottom is different. I followed the first example I saw on the page!